CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+## v1.16.0 (13/06/2023)
+
+**Common**
+* Added `olp::http::CertificateSettings` struct to store custom certificate settings.
+* Added `olp::http::NetworkInitializationSettings` struct containing `olp::http::CertificateSettings` to be passed to `olp::http::CreateDefaultNetwork(..)`.
+* Deprecated `olp::http::CreateDefaultNetwork()` use added `olp::http::CreateDefaultNetwork(..)` that take `olp::http::NetworkInitializationSettings`.
```

Comment:
```suggestion
* Deprecated `olp::http::CreateDefaultNetwork()`.  It will be removed by 05.2024. Use added `olp::http::CreateDefaultNetwork(..)` that take `olp::http::NetworkInitializationSettings` instead.
```

Reply:
Done